app/lib/forms/your_employment.rb

```diff
@@ -0,0 +1,58 @@
+module Forms
+  class YourEmployment < Base
+    QUESTION_NAME = :employment_type
+
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
+
+    def self.permitted_params
+      [QUESTION_NAME]
+    end
+
+    def question
+      OpenStruct.new(
```

Comment:
I'd probably lean towards making this a class if the same (or similar) arguments are going to be used from multiple places.

Reply:
Good suggestion, I've added question type classes that can hold customisation information about different question types automatically.